packages/synapse-react-client/src/components/Markdown/MarkdownUtils.ts

```diff
@@ -0,0 +1,130 @@
+import katex from 'katex'
```

Comment:
Extracted here utilities that were in the MarkdownSynapse class component. No logical changes to this code.